Add `Mapping Global Taiko Showdown 2025: Registrations Open!` news post and wiki page (#13316)
ppy/osu-wiki: news/2025/2025-06-01-mgts-2025-registrations.md

![](/wiki/shared/news/2025-06-01-mgts-2025-registrations/hatching.png)

| Song | [Faxyne](https://soundcloud.com/faxyne) vs. [blobdash](https://soundcloud.com/blobdash) - [Hatching](https://blobdash.bandcamp.com/track/hatching) |
| :-- | :-- |
| **Length** | 2:38 |
| **BPM** | 200 |
| **Note** | Tech pick |

<audio controls>
    <source src="https://assets.ppy.sh/media/news/mgts-2025-hatching.mp3">
</audio>

## Rules

Everything about the contest's rules, procedures, and judging criteria can be found in both the [contest's website](https://gtsosu.com/2025/mgts/home) and its [osu! wiki page](/wiki/Contests/MGTS/2025) — please make sure you and your team go through them thouroughly in order to make sure you end up with a valid submission!

## How do I participate?

First of all, joining the [Discord server](https://discord.com/invite/3mGC3HB) is mandatory for participation, because all important announcements will be communicated there.

Your next step will be to form a team with 2 to 3 members, and have the captain [register the team in the GTS website](https://gtsosu.com/2025/mgts/home).

Once that's done, download one of the two beatmap templates below to start mapping:

> - [Beatmap template 1: ILLUSORYPROGRAM97 + axis::BREAK](https://assets.ppy.sh/media/news/MGTS%202025%20Template%201.osz)

Reviewer: ![image](https://github.com/user-attachments/assets/5391d76b-d3fb-4c44-9534-9c66ab5d1988)

- [Beatmap template 2: ILLUSORYPROGRAM97 + Hatching](https://assets.ppy.sh/media/news/MGTS%202025%20Template%202.osz)

> Make sure to update the beatmap creator's name to **your team's captain's name** in order to have a valid submission.

Reviewer 2: ```suggestion
Make sure to update the beatmap creator's name to **your team captain's name** in order to have a valid submission.
```

> After you're done mapping, have **your team's captain** head over to the [submission page](https://osu.ppy.sh/community/contests/248) to submit your entry. However, don't be too slow about it because you have **29 days** before the submission deadline!

Reviewer 2: ```suggestion
After you're done mapping, have **your team captain** head over to the [submission page](https://osu.ppy.sh/community/contests/248) to submit your entry. However, don't be too slow about it because you have **29 days** before the submission deadline!
```

If you're having any issues with the registration process, please don't hesitate to [send Hivie a message](https://osu.ppy.sh/community/chat?sendto=14102976)!

## Schedule

| Event | Timestamp (UTC) |
| --: | :-- |
| Registration and submission phase | 1 June – 29 June 23:59 |
| Screening phase | *Tentative* |
| Judging phase | 30 June – 26 July |
| Results | 27 July |

The schedule may be subject to change depending on how the contest progresses — we'll be sure to announce any changes in the [Discord server](https://discord.com/invite/3mGC3HB).

> Don't forget to tune in to the [Twitch channel](https://www.twitch.tv/gtsosu) for the results live stream!

Reviewer 2: ```suggestion
Don't forget to tune in to the [Twitch channel](https://www.twitch.tv/gtsosu) for the results livestream!
```
